src/server/cmd/pachctl/cmd/cmd.go

```diff
@@ -124,25 +125,61 @@ This resets the cluster to its initial state.`,
 		}),
 	}
 	var port int
+	var uiPort int
+	var uiWebsocketPort int
 	var kubeCtlFlags string
 	portForward := &cobra.Command{
 		Use:   "port-forward",
 		Short: "Forward a port on the local machine to pachd. This command blocks.",
 		Long:  "Forward a port on the local machine to pachd. This command blocks.",
 		Run: cmdutil.RunFixedArgs(0, func(args []string) error {
-			stdin := strings.NewReader(fmt.Sprintf(`
+
+			var eg errgroup.Group
+
+			eg.Go(func() error {
+				stdin := strings.NewReader(fmt.Sprintf(`
 pod=$(kubectl %v get pod -l app=pachd | awk '{if (NR!=1) { print $1; exit 0 }}')
 kubectl %v port-forward "$pod" %d:650
 `, kubeCtlFlags, kubeCtlFlags, port))
-			fmt.Println("Port forwarded, CTRL-C to exit.")
-			return cmdutil.RunIO(cmdutil.IO{
-				Stdin:  stdin,
-				Stderr: os.Stderr,
-			}, "sh")
+				fmt.Println("Port forwarded, CTRL-C to exit.")
```

Comment:
This is minor but it seems a little ugly to print the `CTRL-C` message in one of the messages and not the others. Why not just print 1 single message for all of them? It's not like the timing of the messages actually correspond to when the ports become available.